### PR TITLE
feat(Tab): add horizontally scrollable Tab.Items

### PR DESCRIPTION
**Pull request by entrymaster**
Feat:
 - added scrollable tab
 - Added few Style Prop
 
Fix: 
- Remove default theme background

### CODE REVIEW

**Comment by entrymaster** [comment]
Scrolling Tabs added with syncing

**Review by entrymaster — COMMENTED** [review]
I have removed some unnecessary code.

File: packages/base/src/Tab/Tab.tsx [hunk 1]
```diff
@@ -14,6 +15,9 @@ export type TabBaseProps = ViewProps & {
   /** Child position index value. */
   value?: number;
 
+  /** Makes Tab Scrolling */
+  tabScrolling?: boolean;
```
Comment (arpitBhalla) [inline packages/base/src/Tab/Tab.tsx:19]: Use `scrollable` instead

File: packages/base/src/Tab/Tab.tsx [hunk 2]
```diff
@@ -32,13 +36,17 @@ export const TabBase: RneFunctionComponent<TabBaseProps> = ({
...
   onChange = () => {},
   indicatorStyle,
   disableIndicator,
   variant,
   ...rest
 }) => {
   const [dim, setDim] = React.useState({ width: 0 });
+  const [childCoords, setChildCoords] = React.useState([]);
+  const [tabXPosition, setTabXPosition] = React.useState(null);
+  const [scrollViewRef, setScrollViewRef] = React.useState(null);
```
Comment (arpitBhalla) [inline packages/base/src/Tab/Tab.tsx:49]: Use `useRef` instead of state

**Review by arpitBhalla — CHANGES_REQUESTED** [review]

**Review by entrymaster — COMMENTED** [review]
OKay

**Pull request merged**